index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio header & headline section</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#home" class="active">Ahmed Adel</a>
+            <a href="#home" class="icon">
+                <i class="fa fa-bars"></i>
+            </a>            
+        </nav>
+
+    </header>
+    <main>
+        <section class="text-content">
+            <h1>Hey There.<br>I'm Ahmed</h1>
+            <h2 class="caption">I’am a Software Developer</h2>
+            <p>I can help you build a product , 
+                feature or website Look through some of my work
+                and experience! If you like what you see and have a project 
+                you need coded, don’t hestiate to contact me.
+            </p>
```

Comment:
- [ ] Your code are well organized however you need to fix your headings and paragraph.
![port11](https://user-images.githubusercontent.com/55635977/129631750-b9b8599b-850f-4c25-a545-caa16bdb134b.png)